### ISSUE TITLE
Feature Request: Links in the README for discussion mailing lists and forums for questions and issues that don't belong on Github

### ISSUE THREAD

**Issue by baconicsynergy** [AmbiML/sparrow-manifest#10]
This project is awesome. You made the right choice choosing seL4 as your microkernel. I want to discuss issues and features with the developers, contributors, and enthusiasts concerning KataOS and Sparrow, but some of my concerns don't belong on these Github pages. Can we direct new folks like me towards discussion forums and mailing lists in the README?

**Comment by jtg-google** [comment]
I think this is a larger question we need to answer for sure. We don't currently have a public discussion forum, and we really should set one up. I'll see what we can do internally to get this ball rolling.

**Comment by sleffler** [comment]
i always assumed any discussion about our code would/could
happen ***@***.*** least until the list owners asked us to move
:)

**Comment by jtg-google** [comment]
Yeah, that's what I think I was assuming as well, though I think maybe it might be worth spinning up an external Google Group, maybe. Need to discuss more.
